lib/registration.pm

```diff
@@ -260,6 +260,11 @@ sub fill_in_registration_data {
             assert_screen('scc-beta-filter-checkbox');
             send_key('alt-i');
         }
+        elsif (!check_screen($modules_needle)) {
+            record_soft_failure('Modules are still beta until GM or FCS');
```

Comment:
I guess you know by now that we try to avoid check_screen with default timeout.
A multi-tag assert_screen with match_has_tag is better.

Even more a record_soft_failure should always have a ticket reference. I see no problem to create a bug in bugzilla and track this one.

I do not buy into any "it is only temporary" argument :)